builder/rootfs/usr/local/src/slugbuilder/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM heroku/cedar:14
 RUN mkdir /app
 RUN addgroup --quiet --gid 2000 slug && \
   useradd slug --uid=2000 --gid=2000 --home-dir /app --no-create-home
-RUN chown -R slug:slug /app
+RUN chown -R 2000:2000 /app
```

Comment:
There was an error where Docker was assigning slug the UID 1000 instead of 2000 in spite of line 5. I think I fixed the problem elsewhere. I will go ahead and change it back... but there was a reason behind using the UID instead of the name.